### PR TITLE
i18n(pt-BR): Add `install/manual.md` page

### PR DESCRIPTION
**Pull request by caioferrarezi**
Portuguese translation for `install/manual.md` page.

### CODE REVIEW

File: src/pages/pt-BR/install/manual.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    color: orange;
+  }
+</style>
+```
+
+## 4. Crie seu primeiro asset estático
+
+Você também irá precisar criar um diretório `public/` para guardar seus assets estáticos. Astro irá sempre incluir estes assets na construção final do seu proejto, você pode referenciá-los com segurança dentro de seus componentes.
+
+Em seu editor de texto, crie um novo arquivo em seu diretório `public/robots.txt`. `robots.txt` é um arquivo simples que é incluídos na maiorias dos sites para dizer aos robôs de busca, como o Google, como tratar o seu site.
```
Comment (Yan-Thomas) [inline src/pages/pt-BR/install/manual.md:90]: Typo `incluídos` -> `incluído`

```suggestion
Em seu editor de texto, crie um novo arquivo em seu diretório `public/robots.txt`. `robots.txt` é um arquivo simples que é incluido na maiorias dos sites para dizer aos robôs de busca, como o Google, como tratar o seu site.
```

**Review by Yan-Thomas — COMMENTED** [review]
Great work. Just found a small typo but the translation is amazing!

**Comment by caioferrarezi** [comment]
> Great work. Just found a small typo but the translation is amazing!

Thank you so much, @Yan-Thomas!

**Comment by Yan-Thomas** [comment]
LGTM! 🚀

**Pull request merged**